fuzz/fuzz_targets/parse_dictionary.rs

```diff
@@ -1,7 +1,18 @@
 #![no_main]
 
 use libfuzzer_sys::fuzz_target;
+use sfv::SerializeValue as _;
 
 fuzz_target!(|data: &[u8]| {
-    let _ = sfv::Parser::from_bytes(data).parse_dictionary();
+    if let Ok(dict) = sfv::Parser::from_bytes(data).parse_dictionary() {
+        let serialized = dict.serialize_value();
+        if dict.is_empty() {
+            assert!(serialized.is_err());
```

Comment:
This was surprising to me - "Why do we expect serialization to fail when the dict is empty?"
I couldn't find why this is the case in #62 
As far as I can tell there's nothing in https://www.rfc-editor.org/rfc/rfc9651.html#name-serializing-a-dictionary that should make it fail.
Could you file a follow-up issue to look into this?